unabridged-works-volume-1/original/14-boston.adoc

```diff
@@ -2119,12 +2200,19 @@ and not so much as an offence because of any difference
 (much less heart-burnings and persecutions);
 but a sweet waiting on the Lord for every one`'s growth in their several ranks and stations.
 
+====
+
 [.blurb]
 === Since my waiting on the LORD for the Presence and Guidance of his Spirit, in the examining the foregoing Grounds and Considerations, there came forth an appendix to JOHN NORTON`'s Book, wherein are laid down some further Grounds by way of justifying of their Proceedings; which, for their Sakes, and likewise on the Behalf of the Truth and People of GOD, I may also say somewhat to.
```

Comment:
Can you get the 3 capitalized words in this blurb and change them to italics (with capitalized first letter)?